src/core/annotation.js

```diff
@@ -920,20 +935,98 @@ class TextWidgetAnnotation extends WidgetAnnotation {
       this.data.maxLen !== null;
   }
 
-  getOperatorList(evaluator, task, renderForms) {
-    if (renderForms || this.appearance) {
-      return super.getOperatorList(evaluator, task, renderForms);
-    }
+  getOperatorList(evaluator, task, renderForms, annotationStorage) {
+    const self = this;
```

Comment:
If arrow functions are used, we shouldn't need variables like `self` because the `this` scope is propagated then.